src/service/video-manager-impl.js

```diff
@@ -538,9 +580,13 @@ class VideoEntry {
       }
```

Comment:
I'll extract this if statement into `loadedVideoVisibilityChanged_`, since `autoplayLoadedVideoVisibilityChanged_` is only called from there.